aksel.nav.no/website/pages/eksempler/primitive-bleed/full.tsx

```diff
@@ -1,39 +1,41 @@
-import { Bleed, Box, HStack } from "@navikt/ds-react";
+import { Bleed, BodyLong, Box, HStack, Page } from "@navikt/ds-react";
 import { withDsExample } from "@/web/examples/withDsExample";
 
 const Example = () => {
   return (
-    <DemoWrapper>
+    <Page.Block width="text">
```

Comment:
Since i can't see what the "container" for top and bottom-sections, it's not immediately obvious for me what marginInline solves here when looking at the example.

Reply:
But in context of the description above the example, this might not actually matter, so can ignore this comment.